scripts/test

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname $0)"/..
+
+echo Running unit tests
+
+# Check if hugepages are configured
+hugepages="$(grep HugePages_Total < /proc/meminfo | awk '{print $2}')"
+if [ -z "$hugepages" ] || [ 1 -gt $hugepages ]
+then
+  echo No hugepages configured on the host for the test
+  exit 1
+fi
+
+mount --rbind /host/dev /dev
+mount --rbind /host/sys /sys
+trap "umount /dev && umount /sys" EXIT
+
+# Do cleanup first
+losetup -D
+trap "losetup -D" EXIT
+
+PACKAGES="$(find . -name '*.go' -print0 | xargs -0 -I{} dirname {} |  cut -f2 -d/ | sort -u | grep -Ev '(^\.$|.git|.trash-cache|vendor|bin)' | sed -e 's!^!./!' -e 's!$!/...!')"
+
+trap "rm -f /tmp/test-disk" EXIT
```

Comment:
nit: `/tmp/test-disk` is created in `spdk_test.go` and it will be used during the test execution. I would suggest deleting it in the test teardown in code rather than this script separately to manage test artifacts in the same place.